src/coffee/import.coffee

```diff
@@ -463,20 +481,9 @@ class Import
     else
       if allUpdateRequests.actions.length
         chunkifiedUpdateRequests = @splitUpdateActionsArray(allUpdateRequests, 500)
-        Promise.all(_.map chunkifiedUpdateRequests, (updateRequest) =>
-          productsUri = createRequestBuilder({ @projectKey })
-            .products
-            .byId(existingProduct.id)
-            .build()
-          request = {
-            uri: productsUri
-            method: 'POST'
-            body: updateRequest
-          }
-          @client.execute request
-        )
-        .then (result) =>
-          @publishProduct(result.body, rowIndex)
+        @updateProductInBatches(existingProduct, chunkifiedUpdateRequests)
+        .then (resProduct) =>
```

Comment:
can this be a bit more descriptive?
I would have gone for `updatedProduct`, since that is the result of the entire operation

Reply:
Updated here 121e257